### DISCUSSION TITLE
New project just coming in to a beta phase!

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#4241] [Show and tell]
Something i have been working on http://stardeck.runasp.net/stardeck **StarDeck** is my "Themodule" - dont think this naming will take on -  a Theme and Module that work in unison.  You literally start with a blank deck, all the new moderen stuff is included; Scroll to, AOS, HTML/CSS designer, Templating and auto generated webparts.

**Comment by rchibnik** [comment]
Can i log in to see ? is there a demo user name? Thanks
